VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
@Haacked My point is that VS 2015 only uses `<ProjectName>.VC.opendb` and `<ProjectName>.VC.VC.opendb` (latter only temporarily while the project is open in IDE) for its new databases. Adding `*.opendb` we simply ignoring *all* files with that extension, it can be user files too. I think it will be best to add `*.VC.VC.opendb` plus `*.VC.opendb` and remove `*.opendb` line.